conda_pip/plugin.py

```diff
@@ -14,3 +14,12 @@ def conda_subcommands():
         action=execute,
         configure_parser=configure_parser,
     )
+
+
+@plugins.hookimpl
+def conda_post_commands():
+    yield plugins.CondaPostCommand(
+        name="conda-pip-ensure-target-env-has-externally-managed",
+        action=ensure_target_env_has_externally_managed,
+        run_for={"install", "create", "update", "remove"},
```

Comment:
This should also cover the various `env` subcommands

Reply:
It's tricky to know how we should handle this in combination with the potential `conda env` hook. I'd rather think about in a different PR so the conversations are well captured separately.